assets/src/blocks/Submenu/SubmenuBlock.js

```diff
@@ -1,158 +1,72 @@
-       *
-       */
-      transforms: {
-        from: [
-          {
-            type: 'shortcode',
-            // Shortcode tag can also be an array of shortcode aliases
-            // This `shortcode` definition will be used as a callback,
-            // it is a function which expects an object with at least
-            // a `named` key with `cover_type` property whose default value is 1.
-            // See: https://simonsmith.io/destructuring-objects-as-function-parameters-in-es6
-            tag: 'shortcake_submenu',
-            attributes: {
-              submenu_style: {
-                type: 'integer',
-                shortcode: function (attributes) {
-                  return Number(attributes.named.submenu_style);
-                }
-              },
-              title: {
-                type: 'string',
-                shortcode: function (attributes) {
-                  return attributes.named.title;
-                }
-              },
-              levels: {
-                type: 'array',
-                shortcode: function (attributes) {
-                  let levels = [];
-                  if (attributes.named.heading1 > 0) {
-                    let level = {
-                      heading: Number(attributes.named.heading1),
-                      link: Boolean(attributes.named.link1) || false,
-                      style: attributes.named.style1 || 'none'
-                    };
-                    levels.push(Object.assign({}, level));
-
-                    if (attributes.named.heading2 > 0) {
-                      let level = {
-                        heading: Number(attributes.named.heading2),
-                        link: Boolean(attributes.named.link2) || false,
-                        style: attributes.named.style2 || 'none'
-                      };
-                      levels.push(Object.assign({}, level));
-
-                      if (attributes.named.heading3 > 0) {
-                        let level = {
-                          heading: Number(attributes.named.heading3),
-                          link: Boolean(attributes.named.link3) || false,
-                          style: attributes.named.style3 || 'none'
-                        };
-                        levels.push(Object.assign({}, level));
-                      }
-                    }
-                  }
-                  return levels;
-                },
-              }
-            },
-          },
-        ]
-      },
       attributes: {
-        submenu_style: {
-          type: 'integer',
-          default: 1
-        },
         title: {
           type: 'string',
+          default: ''
+        },
+        submenu_style: { // Needed for old blocks conversion
+          type: 'integer',
+          default: 0
         },
         levels: {
           type: 'array',
-          default: [ {heading: 0, link: false, style: 'none'}]
+          default: [{ heading: 0, link: false, style: 'none' }]
```

Comment:
When adding this block, heading is at 0 but the `Submenu item` selector in the side panel is on `Heading 2`, which means no header is added to the submenu until we actively select something in `Submenu item` even if it indicates a valid value; maybe we should make the default to heading 2, or add an empty option `Select heading level` for the `SubmenuLevel`